subsys/usb/device/class/audio/audio.c

```diff
@@ -541,6 +556,82 @@ static int handle_fu_mute_req(struct usb_audio_dev_data *audio_dev_data,
 	return -EINVAL;
 }
 
+/**
+ * @brief Handler for feature unit volume control requests.
+ *
+ * This function handles feature unit volume control request.
+ *
+ * @param audio_dev_data USB audio device data.
+ * @param setup          Information about the executed request.
+ * @param len		 Size of the buffer.
+ * @param data		 Buffer containing the request result.
+ * @param evt		 Feature Unit Event info.
+ * @param device	 Device part that has been addressed. Applicable for
+ *			 bidirectional device.
+ *
+ * @return 0 if successful, negative errno otherwise.
+ */
```

Comment:
Sorry, I completely lost track. You can drop the doxygen comments, they are not used here anyway.  The others should also be removed but it is my fault that they are there at all.

Reply:
OK, I have removed the comments.